integration_tests/dbt_project.yml

```diff
@@ -1,7 +1,7 @@
 config-version: 2
 
 name: 'stripe_source_integration_tests'
-version: '0.9.3'
+version: '0.10.0'
```

Comment:
Can you include the source identifiers for the `dispute` and `transfer` seeds in the below section.

lines 12 - 31

Reply:
added